SETUP.md

```
@@ -0,0 +1,334 @@
+# Install backend server dependencies
+cd server && npm install && cd ..
+```
+
+---
+
+## 🔑 Step 2: Get Your Dodo Payments Credentials
+
+### 2.1 Get API Keys
+
+1. Go to [Dodo Payments Dashboard]
+2. Copy your **Test Mode API Key** (starts with `key_test_...`) (https://app.dodopayments.com/developer/api-keys)
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)